opm/input/eclipse/EclipseState/Grid/MULTREGTScanner.hpp

```diff
@@ -76,7 +76,7 @@ namespace Opm {
         }
     };
 
-    typedef std::map< std::pair<int , int> , const MULTREGTRecord * >  MULTREGTSearchMap;
+    typedef std::map< std::pair<int , int> , std::vector<MULTREGTRecord>::size_type>  MULTREGTSearchMap;
```

Comment:
While at it, how about changing to using.

Reply:
> While at it, how about changing to using.

I'll do that when I rebase #3639 on top of this.

Reply:
>> While at it, how about changing to `using`.
>
> I'll do that when I rebase #3639 on top of this.

I changed my mind and have pushed an update that switches to `using` instead of `typedef`.